server/game/cards/01-Core/RisenFromTheSea.js

```diff
@@ -14,7 +14,10 @@ class RisenFromTheSea extends DrawCard {
             handler: context => {
                 context.event.saveCard(context.target);
                 this.controller.attach(this.controller, this, context.target, 'play');
-                this.setCardType('attachment');
+
+                if(this.parent) {
```

Comment:
I'd maybe check `this.controller.canAttach(this, context.target)` instead and move the actual `attach` call inside the the if. Result is the same - at least currently - but it reads better and should be slightly more resilient to any changes made to the attachment system.